packages/liveblocks-react/src/comments/CommentsRoom.ts

```diff
@@ -1,4 +1,26 @@
-/// <reference types="react/experimental" />
+/**
+MIT License
```

Comment:
The revalidation logic is based on [swr](https://swr.vercel.app)'s revalidation implementation. I've added credits here!

Reply:
I'm not sure if the full license is necessary (especially given that the logic is going to evolve greatly over time with the remaining optimizations we have in mind), ~maybe just a normal comment explaining that this implementation is based on SWR and linking to it?~ I see that there's already another comment below linking to SWR directly.

Reply:
Since this isn't literally Vercel's code you're using here, but just borrowing ideas (which is originally the idea from [HTTP RFC 5861](https://datatracker.ietf.org/doc/html/rfc5861)), I don't think this copyright comment is technically appropriate here? I'd suggest replacing the comment with a shout-out or attribution of inspiration to the SWR project, and maybe include a link for reference, but avoid using a copyright.

Reply:
Done!